templates/definition/meter/solaredge-inverter.yaml

```diff
@@ -20,23 +20,20 @@ params:
 render: |
   type: custom
   power:
-    source: modbus
+    source: sunspec
     {{- include "modbus" . | indent 2 }}
     timeout: {{ .timeout }}
   {{- if eq .usage "grid" }}
-    model: sunspec
     subdevice: 1
     value: 203:W # sunspec 3-phase meter power reading
     scale: -1
   {{- else }}
-    model: sunspec
-    value: Power
+    value: Power ## TODO
```

Comment:
@premultiply DCW- aber aus welchem String? Die usage cases sollten wir komplett teilen um es übersichtlicher zu machen.

Reply:
Hier braucht es die Aliase um unterschiedliche Meter Models zu unterstützen.

Auf Model-spezifische feste Einträge zu gehen ist daher keine Lösung.

Reply:
> Hier braucht es die Aliase um unterschiedliche Meter Models zu unterstützen.

Hier gehts ja um den Inverter, nicht um angeschlossene Zähler (usage != grid). Oder hat der selbst keine AC Leistung?

Dann gäbe es:

- 101 (1p int)
- 103 (3p int)
- 111 (1p float)
- 113 (3p float)

Brauchen wir wirklich alle? Dann würde ich versuchen, einen Selektor zu implementieren. Welches Modell existiert lässt sich ja herausfinden.

Reply:
101 und 103 sind mir gängig und müssten mindestens automatisch adressiert werden.
111 und 113 habe ich noch nicht bewusst gesehen. Aber man guckt ja nur im Problemfall genauer auf das Modell...

Reply:
Ich habs rausgeschmissen. Hier gehts ja um den else Zweig. Grid war schon separat, PV auch. Das kommt also nie zur Anwendung.